README.md

```diff
@@ -26,13 +26,16 @@ ember install ember-frost-object-browser
 
 ### Action Bar
 
-| Attribute        | Type          | Value | Description                                         |
-| ---------------- | ------------- | ----- | --------------------------------------------------- |
-| controls         | `array`       |       | Controls that will be available in the action bar   |
-| isLoading        | `boolean`     | false | **default** - Action bar is not in a loading state  |
-|                  |               | true  | Action bar is in a loading state                    |
-| loadingText      | `string`      |       | Text that appears beside the loading animation      |
-| selectedItems    | `array`       |       | List of items that are currently selected           |
+| Attribute             | Type          | Value | Description                                                 |
+| --------------------- | ------------- | ----- | ----------------------------------------------------------- |
+| controls              | `array`       |       | Controls that will be available in the action bar           |
```

Comment:
`controls` type is `EmberComponent[]|object`

Reply:
`Ember.Component[]|object`